changelogs/fragments/bugix_l3_interface.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
This should be a major release I suppose!

Reply:
Ack